radis/extractions/views.py

```
@@ -194,6 +450,129 @@ class ExtractionJobResumeView(ExtractionsLockedMixin, AnalysisJobResumeView):
+
+            context = {
+                "generated_query": generated_query,
+                "query_metadata": metadata,
+                "output_fields_count": len(temp_fields),
+                "error": None if metadata.get("success") else "Query generation failed",
+            }
+
+        except (RuntimeError, ValueError) as e:
+            logger.error(f"Error during async query generation: {e}", exc_info=True)
+            context = {
+                "error": f"Error generating query: {str(e)}",
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)